### PR TITLE
hack/bin/integration-cleanup.sh: Use jq magic instead of bash

### PR DESCRIPTION
**Pull request by akshaymankar**
For some reason the the bash expression to calculate time fails in CI and helm
now supports outputting json, so I swapped bash magic with jq magic. It more
robust, hopefully it is also more understandable.

### CODE REVIEW

File: hack/bin/filter-old-releases.jq [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.[]
+  | .parsedUpdatedTime = ( .updated
+                         | sub("(?<time>.*)\\.[\\d]+(?<tz>.*)"; "\(.time)\(.tz)")
+                         | strptime("%Y-%m-%d %H:%M:%S %z %Z")
+                         | mktime
+                         )
+  | select ( .parsedTime < (now - 3 * 60 * 60))
+  | [.name, .namespace]
+  | @tsv
```
Comment (lucendio) [inline hack/bin/filter-old-releases.jq:9]: question: why tabs?

**Review by lucendio — APPROVED** [review]
LGTM

File: hack/bin/integration-cleanup.sh [hunk 2]
```diff
@@ -4,31 +4,15 @@
...
-
-IFS=$'\n'
-for release in $(helm list -A | grep test-); do
-  name=$(echo "$release" | awk '{print $1}')
-  namespace=$(echo "$release" | awk '{print $2}')
-  date_str=$(echo "$release" | awk '{print $4" "$5" "$6}')
-  date_s=$(date_seconds "$date_str")
-  diff=$(( $date_now - $date_s ))
-  if [ $diff -ge 7200 ]; then
-    echo "test release '$name' older than 2 hours; deleting..."
```
Comment (fisx) [inline hack/bin/integration-cleanup.sh:29]: wanna keep that echo statement in some modified form?

File: hack/bin/filter-old-releases.jq (hunk 1 above)
Comment (akshaymankar) [reply hack/bin/filter-old-releases.jq:9]: I think jq can either output a json array, a csv or a tsv. Out of these I thought tsv was the most compatible with bash as I can `awk '{print $1}'`. Do you have a better idea?

**Review by jschaul — APPROVED** [review]
There were some problems with the script still

- it errors if `releases` is an empty string:

```
./hack/bin/integration-cleanup.sh
test release '' older than 2 hours; deleting...
Error: uninstall: Release name is invalid: 
```

I added a commit which should fix that.

**Comment by akshaymankar** [comment]
The tests are failing because this incorrectly deletes releases deployed less than 2 hours ago, I will look into that.

**Comment by akshaymankar** [comment]
Maybe I was a fool to assume jq is safer than bash :cry: https://github.com/wireapp/wire-server/pull/1432/commits/c225f41a6fe66623a53152a39262648f2efe145a

**Comment by jschaul** [comment]
All green now, it seems!

File: hack/bin/filter-old-releases.jq (hunk 1 above)
Comment (lucendio) [reply hack/bin/filter-old-releases.jq:9]: I just stumbled upon it and was curious, as `FS` usually defaults to *space*.

**Pull request merged**